pkg/backup/item_collector_test.go

```diff
@@ -48,3 +49,32 @@ func TestSortCoreGroup(t *testing.T) {
 		assert.Equal(t, expected[i], r.Name)
 	}
 }
+
+func TestSortOrderedResource(t *testing.T) {
+	log := logrus.StandardLogger()
```

Comment:
Thanks.  I'll address it in the next one.  I plan to revisit this feature to enhance it with regular expression instead of simply exact name matching.